packages/@react-spectrum/button/src/Button.tsx

```diff
@@ -78,10 +78,9 @@ function Button<T extends ElementType = 'button'>(props: SpectrumButtonProps<T>,
   let [isProgressVisible, setIsProgressVisible] = useState(false);
   let backupButtonId = useId();
   let buttonId = buttonProps.id || backupButtonId;
-  let spinnerId = useId();
-  let textId = useId();
   let iconId = useId();
-  let auxLabelId = useId();
+  let textId = useId();
+  let spinnerId = useId();
```

Comment:
I'm weird, and felt compelled to generate ids in the order they appear in the DOM, which helps make debugging in the Web Inspector a little easier.